rust/connlib/clients/android/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 
 [lib]
 name = "connlib"
-crate-type = ["cdylib"]
+crate-type = ["lib", "cdylib"]
```

Comment:
Otherwise rustc complains that `cdylib` is not compatible with `musl` targets when building other packages.